.travis.yml

```diff
@@ -7,8 +7,10 @@ python:
 install:
   - pip install -r dev-requirements.txt
   - pip install -e .
+before_script:
+  - docker-compose up &
```

Comment:
This will make tests flaky since `&` doesn't guarantee that the containers are up before proceeding. What you want is `-d` instead, which will wait until the containers are up and *then* detach.